nilearn/reporting/_get_clusters_table.py

```diff
@@ -267,6 +267,10 @@ def get_clusters_table(stat_img, stat_threshold, cluster_threshold=None,
                            Rows corresponding to subpeaks will not have a value
                            in this column.
         ================== ====================================================
+        
+    label_maps : :obj:`list`
+        List of Niimg-like objects of cluster label maps.
+        If two_sided==True, first and second maps correspond to positive and negative tails.
```

Comment:
This is a breaking change since `get_clusters_table` is a public function. All user code expecting a single DataFrame as return value will break because it will receive a tuple instead.
A possible solution is to add an additional `return_label_maps` parameter to the function with a default value of False and a Warning explaining that the parameters will default to True in a couple release, and be removed in two more releases.

Reply:
> This is a breaking change since get_clusters_table is a public function. All user code expecting a single DataFrame as return value will break because it will receive a tuple instead.
A possible solution is to add an additional return_label_maps parameter to the function with a default value of False

Good point. I just added a commit to make a `return_label_maps` flag for the function. This also means that the instances in the tutorials do not have to be changed at all.

> Warning explaining that the parameters will default to True in a couple release, and be removed in two more releases.

Is there a function you can think of that currently does this that I can use as a style guide?

Reply:
See e.g. FutureWarning occurrence in in `nilearn/datasets/struct.py`